.travis.yml

```
@@ -0,0 +1,12 @@
+language: python
+matrix:
+    include:
+        - python: 3.6
```

And zero indentation for sequence items
```suggestion
    - python: 3.6
```